lib/auth/auth.ts

```diff
@@ -1,6 +1,6 @@
 import * as crypto from 'crypto';
-import { Logger } from 'werelogs';
-import errors from '../errors';
+import { RequestLogger } from 'werelogs';
```

Comment:
```suggestion
import type { RequestLogger } from 'werelogs';
```

For type only import this can be interesting as the import is discarded after transpiling